NAMESPACE

```diff
@@ -11,6 +11,8 @@ export(empty_block)
 export(fit_TIRT_lavaan)
 export(fit_TIRT_mplus)
 export(fit_TIRT_stan)
+export(get_TIRT_params_stan)
```

Comment:
How would you recommend we integrate this functionality into the `predict` method?